src/helpers/parameters/duplication.js

```diff
@@ -4,13 +4,24 @@
  * @returns {array} an array of errors or an empty array if no error
```

Comment:
why is this file named duplication?

Reply:
@e-liyai I renamed the file to `checkDuplication.js`